static/src/stylesheets/module/facia-garnett/_container.scss

```diff
@@ -164,8 +164,6 @@ $header-image-size-desktop: 100px;
         color: inherit;
 
         &:hover {
-            color: $news-main;
-            text-decoration: none;
             border-bottom: 1px solid $news-main;
```

Comment:
I believe this border bottom can be deleted as well. It seems to be not applied or have no visible impact when it is applied.

Reply:
This may suggest that those links could have been styled like body text links in articles in the past (fancy, take a look). Safe to remove as matching thrasher, if Zef agrees, is a sensible thing.